template/deploy/helm/[[product]]-operator/templates/deployment.yaml.j2

```diff
@@ -44,6 +44,11 @@ spec:
           env:
             - name: KAFKA_BROKER_CLUSTERROLE
               value: {{ .Release.Name }}-kafka-broker-clusterrole
+{[% endif %}]
+{[% if operator.product_string in ['opa'] %}]
+          env:
+            - name: OPA_BUILDER_CLUSTERROLE
+              value: {{ .Release.Name }}-opa-builder-clusterrole
```

Comment:
nit: should this not be called bundle-builder ?

Reply:
Yeah it should.

Reply:
Done.